toolchain/sem_ir/formatter.cpp

```diff
@@ -948,23 +961,10 @@ class FormatterImpl {
   auto FormatArg(FloatKind k) -> void { k.Print(out_); }
 
   auto FormatArg(ImportIRId id) -> void {
-    if (!id.is_valid()) {
-      out_ << id;
-      return;
-    }
-    const auto& import_ir = *sem_ir_.import_irs().Get(id).sem_ir;
-    if (import_ir.package_id().is_valid()) {
-      out_ << import_ir.identifiers().Get(import_ir.package_id());
+    if (id.is_valid()) {
+      out_ << GetImportIRLabel(id);
     } else {
-      out_ << "Main";
-    }
-    out_ << "//";
-    CARBON_CHECK(import_ir.library_id().is_valid());
-    if (import_ir.library_id() == LibraryNameId::Default) {
-      out_ << "default";
-    } else {
-      out_ << import_ir.string_literal_values().Get(
-          import_ir.library_id().AsStringLiteralValueId());
+      out_ << id;
     }
```

Comment:
Optional idea, I'm happy with either form though:

```suggestion
    out_ << (id.is_valid() ? GetImportIRLabel(id) : id);
```

Reply:
toolchain/sem_ir/formatter.cpp:964:28: error: incompatible operand types ('std::string' (aka 'basic_string<char>') and 'ImportIRId')
    out_ << (id.is_valid() ? GetImportIRLabel(id) : id);
                           ^ ~~~~~~~~~~~~~~~~~~~~   ~~

Reply:
:sigh: right, C++ makes that tricky. Oh well (and thanks).